src/workers/squadFeatureAccess.ts

```diff
@@ -0,0 +1,38 @@
+import { messageToJson, Worker } from './worker';
+import { ChangeObject } from '../types';
+import { SourceMember, Feature, FeatureType } from '../entity';
+import { TypeOrmError } from '../errors';
+
+interface Data {
+  sourceMember: ChangeObject<SourceMember>;
+}
+
+const worker: Worker = {
+  subscription: 'api.squad-feature-access',
+  handler: async (message, con, logger): Promise<void> => {
+    const { sourceMember: member }: Data = messageToJson(message);
+    try {
+      await con.getRepository(Feature).insert({
+        feature: FeatureType.Squad,
+        userId: member.userId,
+      });
+    } catch (err) {
+      console.log(err.code);
+      logger.error(
```

Comment:
This needs to be logged after the if because if it already exists so there's no really an error